.github/chronographer.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This is a config for a GitHub App that is able to help with checking whether a change fragment is present in a given PR. It allows skipping this requirement by labeling a PR and for PRs created by certain robots. It needs to be installed separately from the PR but I've copied the config just in case.